setup.py

```diff
@@ -1,11 +1,34 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import absolute_import
+
+import os
+
 from setuptools import find_packages
 from setuptools import setup
 
+
+def read_requirements(file_):
+    lines = []
+    with open(os.path.join(os.path.dirname(__file__), file_)) as f:
```

Comment:
Maybe you have to move this outside the function, like https://github.com/mozilla/libmozdata/blob/master/setup.py#L12.